### PR TITLE
bpo-39417: Fix broken link to guide to building venvs

### PR DESCRIPTION
**Pull request by j9ac9k**
# Fix broken link in python packaging user guide

```
[bpo-39417](https://bugs.python.org/issue39417): Fix broken link to "See also: Python Packaging User Guide: Creating and using virtual environments"
```

https://bugs.python.org/issue39417

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@j9ac9k

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Comment by j9ac9k** [comment]
I have signed the CLA and am waiting the 1 US Business Day before checking the status on the `check-python-cla` website

**Comment by ammaraskar** [comment]
Thanks for your contribution @j9ac9k! Could you put the bugs.python.org issue number in the commit message as shown here: https://devguide.python.org/pullrequest/#making-good-commits

**Comment by j9ac9k** [comment]
sorry for the multiple forced commits, I got a bit configured in the docs with what the format should be of the git commit msg; git commit msg should now be correct.

**Review by ammaraskar — APPROVED** [review]
LGTM once CLA is approved.

**Comment by bonitobird** [comment]
At PyCascades Sprint, this is good to go. @willingc Please review.

**Review by gvanrossum — APPROVED** [review]

**Pull request merged**
